Source/WarpX.H

```diff
@@ -136,112 +138,160 @@ public:
     //
     // Functions used by implicit solvers
     //
-    void ImplicitPreRHSOp ( amrex::Real cur_time,
-                            amrex::Real a_full_dt,
-                            int         a_nl_iter,
-                            bool        a_from_jacobian );
+    void ImplicitPreRHSOp (amrex::Real cur_time, amrex::Real a_full_dt,
+                           int a_nl_iter, bool a_from_jacobian);
     void SaveParticlesAtImplicitStepStart ();
     void FinishImplicitParticleUpdate ();
-    void SetElectricFieldAndApplyBCs ( const WarpXSolverVec& a_E, amrex::Real a_time );
-    void UpdateMagneticFieldAndApplyBCs ( ablastr::fields::MultiLevelVectorField const& a_Bn,
-                                          amrex::Real a_thetadt, amrex::Real start_time );
-    void SpectralSourceFreeFieldAdvance ( amrex::Real start_time);
-    void FinishMagneticFieldAndApplyBCs ( ablastr::fields::MultiLevelVectorField const& a_Bn,
-                                          amrex::Real a_theta, amrex::Real a_time );
-    void FinishImplicitField ( const ablastr::fields::MultiLevelVectorField& Field_fp,
-                               const ablastr::fields::MultiLevelVectorField& Field_n,
-                               amrex::Real theta );
-    void ImplicitComputeRHSE (         amrex::Real dt, WarpXSolverVec& a_Erhs_vec);
-    void ImplicitComputeRHSE (int lev, amrex::Real dt, WarpXSolverVec& a_Erhs_vec);
-    void ImplicitComputeRHSE (int lev, PatchType patch_type, amrex::Real dt, WarpXSolverVec& a_Erhs_vec);
-
-    MultiParticleContainer& GetPartContainer () { return *mypc; }
-    MultiFluidContainer& GetFluidContainer () { return *myfl; }
-    ElectrostaticSolver& GetElectrostaticSolver () {return *m_electrostatic_solver;}
-    [[nodiscard]] HybridPICModel * get_pointer_HybridPICModel () const { return m_hybrid_pic_model.get(); }
-    MultiDiagnostics& GetMultiDiags () {return *multi_diags;}
-    ParticleBoundaryBuffer& GetParticleBoundaryBuffer () { return *m_particle_boundary_buffer; }
-    amrex::Vector<std::array< std::unique_ptr<amrex::iMultiFab>,3 > >& GetEBUpdateEFlag() { return m_eb_update_E; }
-    amrex::Vector<std::array< std::unique_ptr<amrex::iMultiFab>,3 > >& GetEBUpdateBFlag() { return m_eb_update_B; }
-    amrex::Vector< std::unique_ptr<amrex::iMultiFab> > const & GetEBReduceParticleShapeFlag() const { return m_eb_reduce_particle_shape; }
+    void SetElectricFieldAndApplyBCs (const WarpXSolverVec& a_E,
```

Comment:
Presumably these changes are being made because of the line length limit. I've always preferred longer lines which, to me, makes this type of code easier to read and more compact. What is the maximum line length?

Reply:
The LLVM configuration file sets 
```
ColumnLimit:     80
```
so it's probably 80 at the moment.